beep/__init__.py

```diff
@@ -72,7 +75,16 @@
 # output and format
 if 'CloudWatch' in config[ENVIRONMENT]['logging']['streams']:
     if ENVIRONMENT == "stage":
-        hdlr = watchtower.CloudWatchLogHandler(log_group='/stage/beep/services')
+        for _ in range(MAX_RETRIES):
+            try:
+                hdlr = watchtower.CloudWatchLogHandler(log_group='/stage/beep/services')
+            except NoCredentialsError:
+                time.sleep(10)
+                continue
+            else:
+                break
+        else:
+            raise NoCredentialsError
```

Comment:
Just for my understanding, does the retry logic actually work a fraction of the time? If not, how does it help handle the credentials error?

Reply:
When we tried this with the data-syncer, waiting 10 sec and retrying seemed to work almost 100% of the time. No idea why exactly, but my guess is the script is starting before the container can get the credentials from the role